app/helpers/application_helper.rb

```diff
@@ -41,8 +41,8 @@ def errors_for(object, message = nil)
   # see https://github.com/rails/jquery-ujs/wiki/Manual-installing-and-Rails-2
   def csrf_meta_tags
     if protect_against_forgery?
-      out = %(<meta name="csrf-param" content="%s"/>\n)
-      out << %(<meta name="csrf-token" content="%s"/>)
+      out = %(<meta name="csrf-param" content="%s"/>\n) # rubocop:disable Style/FormatStringToken
+      out << %(<meta name="csrf-token" content="%s"/>) # rubocop:disable Style/FormatStringToken
```

Comment:
Why not add this in `rubocop.yml` ?

Reply:
Because the content of this is a crsf token.